packages/ERTP/src/types.js

```diff
@@ -261,10 +261,21 @@
  *
  * @property {() => Brand} getAllegedBrand Get the alleged Brand for this Purse
  *
- * @property {() => Amount} getCurrentAmount
- * Get the amount contained in this purse.
+ * @property {() => Promise<Amount>} getCurrentAmount
+ * at-deprecated Use `getRecentAmount` instead.
+ * TODO How to I tell jsdoc something in this position is deprecated?
```

Comment:
Note: I need advice here as explained in the TODO comment.

Reply:
The JSDoc example has it applied to a single function. Would it work if you pulled this function out? https://jsdoc.app/tags-deprecated.html

Reply:
How would I pull this function out when it is part of an object that implements a declared interface? I just don't know jsdoc well enough to understand what kinds of refactoring I can do on these declarations.

Reply:
Just for the record: I can't figure out how to deprecate a single property.  Googling around implies it isn't possible with current JSDoc, as the `@deprecation` tag is just for a specific symbol, not a property.